etc/ssdlc_compliance_report.md

```diff
@@ -0,0 +1,40 @@
+# MongoDB C++ Driver SSDLC Compliance Report
+
+## Release Creator
+
+- See [C/CXX Release Info](https://docs.google.com/spreadsheets/d/1yHfGmDnbA5-Qt8FX4tKWC5xk9AhzYZx1SKF4AD36ecY/edit?usp=sharing).
+
+## Process Document
+
+- Not available. <!-- CXX-3007: replace with link to public-facing document once available. -->
+
+## Tool used to track third party vulnerabilities
+
+- See [Silk](https://us1.app.silk.security/inventory/asset-group/mongodb____DedupedAssetGroup____60640b8853771efe3af5f78ea37af5d1cdd190df).
+- See [C++ Driver - SSDLC Reports](https://drive.google.com/drive/folders/1q9RI55trFzHlh8McALSIAbT6ugyn8zlO) for release-specific reports.
+
+## Third-Party Dependency Information
+
+- See [etc/augmented.sbom.json](https://github.com/mongodb/mongo-cxx-driver/blob/master/etc/augmented.sbom.json) within the release tarball.
+- See [etc/third_party_vulnerabilities.md](https://github.com/mongodb/mongo-cxx-driver/blob/master/etc/third_party_vulnerabilities.md) within the release tarball.
+- See [C++ Driver - SSDLC Reports](https://drive.google.com/drive/folders/1q9RI55trFzHlh8McALSIAbT6ugyn8zlO) for release-specific reports.
+
+## Static Analysis Findings
+
+- See [C++ Driver - SSDLC Reports](https://drive.google.com/drive/folders/1q9RI55trFzHlh8McALSIAbT6ugyn8zlO) for release-specific reports.
+
+## Security Testing Report
+
+- See [Driver Security Testing Summary](https://docs.google.com/document/d/1y2K_RY4GZVXpQvv4JH_35mSzFRTawNJ3mibpvSBU8H0/edit?usp=sharing). Available upon request for approved partners (contact Rishabh Bisht \<rishabh.bisht@mongodb.com\>).
```

Comment:
I expect the contact information was motivated [by this conversation](https://mongodb.slack.com/archives/C05N744KWQ4/p1717092734498109). However, I am mildly concerned about putting a direct e-mail (may be a target for spam). I expect customers have existing means to contact support. I suggest using wording matching [minimal.md](https://gist.github.com/rtimmons/8253d1dfcc5f233bdb2d9c73b8e5c129#file-minimal-md):

```suggestion
- See [Driver Security Testing Summary](https://docs.google.com/document/d/1y2K_RY4GZVXpQvv4JH_35mSzFRTawNJ3mibpvSBU8H0/edit?usp=sharing) (internal). Available as needed from the MongoDB C++ Driver team.
```

Reply:
Done.